src/Shared/Recruit.Vacancies.Client/Application/Communications/ParticipantResolverPlugins/ParticipantResolverPluginHelper.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using System.Linq;
+using Communication.Types;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+
+namespace Esfa.Recruit.Vacancies.Client.Application.Communications.ParticipantResolverPlugins
+{
+    public static class ParticipantResolverPluginHelper
+    {
+        public static IEnumerable<CommunicationUser> ConvertToCommunicationUser(List<User> user, string primaryUserIdamsId)
```

Comment:
Pluralize function name and parameter `user`

ConvertToCommunicationUser**s**(List<User> user**s**, string primaryUserIdamsId)